src/components/ComputeModal.js

```diff
@@ -304,11 +307,15 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
       await Ajax().Disks.disk(googleProject, currentPersistentDiskDetails.name).delete()
     }
     if (shouldUpdateRuntime) {
-      await Ajax().Runtimes.runtime(googleProject, currentRuntimeDetails.runtimeName).update({ runtimeConfig })
+      await Ajax().Runtimes.runtime(googleProject, currentRuntimeDetails.runtimeName).update({
+        runtimeConfig,
+        autopauseThreshold: computeConfig.autopauseThreshold
```

Comment:
this seems like it is not needed. It's added to the runtimeConfig object on line 266, it shouldn't need to also be a top-level field